src/Searchable/DefaultImportSource.php

```diff
@@ -66,7 +66,7 @@ private function model()
     private function newQuery()
     {
         $query = $this->model()->newQuery();
-        $softDelete = config('scout.soft_delete', false);
+        $softDelete = forward_static_call([$this->className, 'usesSoftDelete']) && config('scout.soft_delete', false);
```

Comment:
`forward_static_call` - [This function must be called within a method context ](https://www.php.net/manual/en/function.forward-static-call.php) I'm not familiar with `forward_static_call` so I'm not sure I understand right what does it mean, maybe we can't use this function here. Could you try another way?
```
$className = $this->className;
$softDelete = $className::usesSoftDelete() && config('scout.soft_delete', false);

```

Reply:
The function `forward_static_call` works, no matter what the test says. But the test doesn't understand the context I guess and therefore it doesn't pass.

No worries, if the other syntax also does work it is fine by me!

Let's test it again...